### PR TITLE
Optimize imports

### PR DESCRIPTION
**Pull request by Matocolotoe**
This just removes many unused imports.

*Note : the diffs of ExprLastSpawnEntity and BukkitLoggerFilter seem broken, even though I literally copied back the file from the master branch after making changes I wanted to revert...*

### CODE REVIEW

File: src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java [hunk 1]
```diff
@@ -1,108 +1,108 @@
-/**
```
Comment (Wealthyturtle) [inline src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java:1]: Line endings need to be fixed for this.

File: src/main/java/ch/njol/skript/log/BukkitLoggerFilter.java [hunk 2]
```diff
@@ -1,61 +1,61 @@
-/**
```
Comment (Wealthyturtle) [inline src/main/java/ch/njol/skript/log/BukkitLoggerFilter.java:1]: Same here

**Comment by Wealthyturtle** [comment]
Other than those, it looks good though!

**Comment by Matocolotoe** [comment]
Thanks ! Will try to fix these endings ASAP

**Comment by Matocolotoe** [comment]
Bored of trying to fix line endings, going to redo everything.

**Pull request closed without merging**